prow/knative/cluster/monitoring/secrets/alertmanager-prow_secret.yaml

```diff
@@ -21,7 +21,7 @@ stringData:
       group_wait: 30s
       group_interval: 10m
       repeat_interval: 4h
-      receiver: 'no-op-alerts'
+      receiver: 'email-warnings'
```

Comment:
This is going to be really noisy since we have a number of alerts that are low priority or inactionable. See the #prow-alerts channel on the k8s slack to see what I mean. Please don't start routing these alerts to us until we've reduced the noise.

Reply:
sg, removed